MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2125,10 +2125,11 @@ public ShoppingList goShopping(ShoppingList sList) {
      * @return true if the campaign can pay for the acquisition; false if it cannot. 
      */
     public boolean canPayFor(IAcquisitionWork acquisition) {
-        if((acquisition instanceof UnitOrder && getCampaignOptions().payForUnits()) 
-                ||(acquisition instanceof Part && getCampaignOptions().payForParts()) 
-                && getFunds() < acquisition.getBuyCost()) {
-            return false;
+    	//SHOULD we check to see if this acquisition needs to be paid for
+        if( (acquisition instanceof UnitOrder && getCampaignOptions().payForUnits()) 
+                ||(acquisition instanceof Part && getCampaignOptions().payForParts()) ) {
+        	//CAN the acquisition actually be paid for
+            return !(getFunds() < acquisition.getBuyCost());
```

Comment:
This would be easier to read as: `return getFunds() >= acquisition.getBuyCost();`